cron.go

```diff
@@ -156,29 +156,27 @@ func parseCronField(fieldIdx int, fieldStr string) (cronField, error) {
 // - Command section
 //
 // - Text representation of the fields "month" and "day of week"
-func New(expr string) (CronMask, error) {
+func New(expr string) (*CronMask, error) {
```

Comment:
I know it's a matter of taste, but I like more the `NewCronMask` style of naming `New`.
Please ignore if you think `New` is better.

Reply:
What I like about having `New` is mostly from user's perspective. To me `cronmask.New` feels better than `cronmask.NewCronMask` as it is already implicit if the library is doing only one thing. What do you think?

Reply:
makes also sense to me.